libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/EmitEvent.cs

```diff
@@ -33,7 +33,7 @@ public EmitEvent(string eventName = null, object eventValue = null, bool bubble
         /// <summary>
         /// Gets or sets the object to send with the event
```

Comment:
>object [](start = 29, length = 6)

"Gets or sets the memory property path to use to get the value to send as part of the event"